<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignManager</class>
 <widget class="QWidget" name="AssignManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(31, 13, 14);
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding:15px 10px;
border:none;
border-radius:15px;</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="nameComboBox">
      <property name="styleSheet">
       <string notr="true">border:1px solid black;</string>
      </property>
      <property name="placeholderText">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="bookNameLabel">
      <property name="text">
       <string>Book Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="bookNameComboBox">
      <property name="styleSheet">
       <string notr="true">border:1px solid black;</string>
      </property>
      <property name="placeholderText">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="countLineEdit">
      <property name="styleSheet">
       <string notr="true">border:1px solid black;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(247, 247, 247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Assign Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookCountLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Technical</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Non-Technical</string>
   </property>
  </widget>
  <widget class="QFrame" name="formFrame_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding:15px 10px;
border:none;
border-radius:15px;</string>
   </property>
   <layout class="QFormLayout" name="formLayout_3"/>
  </widget>
  <zorder>formFrame_2</zorder>
  <zorder>formGroupBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>bookCountLabel</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
